### PR TITLE
Code review

### PR DESCRIPTION
**Pull request by ViniciusCampanha**

### CODE REVIEW

**Comment by ViniciusCampanha** [comment]
jeinkins rebuild

**Comment by ViniciusCampanha** [comment]
jenkins rebuild

File: src/terrama2/services/alert/core/Report.cpp [hunk 1]
```diff
@@ -303,7 +303,9 @@ double terrama2::services::alert::core::Report::retrieveMaxValue() const
     throw ReportException() << ErrorDescription(errMsg);
   }
 
-  return dataSet_->getRaster(pos)->getBand(0)->getMaxValue(true).real();
+  int band = std::stoi(alert_->riskAttribute);
```
Comment (janosimas) [inline src/terrama2/services/alert/core/Report.cpp]: Try and catch to emit a terrama2 exception

File: src/terrama2/services/alert/core/Report.cpp [hunk 2]
```diff
@@ -334,7 +336,9 @@ double terrama2::services::alert::core::Report::retrieveMinValue() const
     throw ReportException() << ErrorDescription(errMsg);
   }
 
-  return dataSet_->getRaster(pos)->getBand(0)->getMinValue(true).real();
+  int band = std::stoi(alert_->riskAttribute);
```
Comment (janosimas) [inline src/terrama2/services/alert/core/Report.cpp]: Try and catch to emit a terrama2 exception

File: src/terrama2/services/alert/core/Report.cpp [hunk 3]
```diff
@@ -365,7 +369,9 @@ double terrama2::services::alert::core::Report::retrieveMeanValue() const
     throw ReportException() << ErrorDescription(errMsg);
   }
 
-  return dataSet_->getRaster(pos)->getBand(0)->getMeanValue().real();
+  int band = std::stoi(alert_->riskAttribute);
```
Comment (janosimas) [inline src/terrama2/services/alert/core/Report.cpp]: Try and catch to emit a terrama2 exception

**Pull request merged**